### PR TITLE
`cat` concatenates `ReplicatedSharedTensor`

### PR DESCRIPTION
**Pull request by Timo9Madrid7**
## Description
- refer to issue #324
- `cat` can concatenates `ShareTensor`, but it is not workable for `ReplicatedSharedTensor`.
- added `cat_replicatedShare_tensor` in *apy.py* and *static.py* for the concatenation of `ReplicatedSharedTensor`.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: src/sympc/tensor/static.py [hunk 1]
```diff
@@ -117,6 +121,27 @@ def cat_share_tensor(session_uuid_str: str, *shares: Tuple[ShareTensor]) -> Shar
...
+        shares (Tuple[ShareTensor]): Replicated shares of the tensors to be concatenated.
+
+    Returns:
+        ReplicatedSharedTensor: Respective replicated shares after concatenation
+    """
+    
+    session = get_session(session_uuid_str)
+    result = ReplicatedSharedTensor(session_uuid=UUID(session_uuid_str), config=session.config)
+    
+    cat_result = [torch.tensor([]).type(torch.LongTensor) for _ in range(len(shares[0].shares))]
```
Comment (gmuraru) [inline src/sympc/tensor/static.py:138]: MAYBE: Could you save the `len(shares[0].shares)` in a variable and then use it here and on line 140?

**Comment by gmuraru** [comment]
Only one comment! I think you also need to run the `black` formatted and then we are ready to merge this :rocket:

File: src/sympc/tensor/static.py (hunk 1 above)
Comment (Timo9Madrid7) [reply src/sympc/tensor/static.py]: Yes, nice catch, that will more efficient. 👍

**Comment by Timo9Madrid7** [comment]
> Only one comment! I think you also need to run the `black` formatted and then we are ready to merge this 🚀

Ok, I have applied `black` formatter for `api.py` and `tensor/static.py`.

**Comment by gmuraru** [comment]
Ah..you also need to use the `isort`.
The better solution is to install the `pre-commit` hooks and they should do the job!

**Comment by Timo9Madrid7** [comment]
> Ah..you also need to use the `isort`. The better solution is to install the `pre-commit` hooks and they should do the job!

Alright! These two files have been checked and passed those tests. Thank you for guiding me.
```
$ pre-commit run --files src/sympc/api.py src/sympc/tensor/static.py
```

**Comment by gmuraru** [comment]
> > Ah..you also need to use the `isort`. The better solution is to install the `pre-commit` hooks and they should do the job!
> 
> Alright! These two files have been checked and passed those tests. Thank you for guiding me.
> 
> ```
> $ pre-commit run --files src/sympc/api.py src/sympc/tensor/static.py
> ```

No worries!

**Review by gmuraru — APPROVED** [review]
LGTM!
